oak/server/storage/storage_node.cc

```diff
@@ -110,19 +111,24 @@ oak::StatusOr<std::unique_ptr<oak::encap::GrpcResponse>> StorageNode::ProcessMet
       return absl::Status(absl::StatusCode::kInvalidArgument, "Failed to parse request");
     }
     // TODO(#449): Check integrity policy for item.
-    std::string item;
-    write_req.item().SerializeToString(&item);
-    OAK_RETURN_IF_ERROR(storage_processor_.Write(write_req.storage_name(), write_req.item().name(),
-                                                 item, write_req.transaction_id()));
+    asylo::CleansingVector<uint8_t> item(write_req.item().ByteSizeLong());
```

Comment:
Is there a reason for using asylo::CleansingVector<uint8_t> here instead of CleansingBytes?

Reply:
Nope, just missed it when I added the alias &ndash; fixed.